### PR TITLE
can make https requests over insecure connections

### PR DESCRIPTION
**Pull request by iamemilio**
- allows users to send insecure/self-signed https messages
@danmcp @rudolphpienaar

### CODE REVIEW

File: bin/pfurl [hunk 1]
```diff
@@ -160,8 +160,15 @@ parser.add_argument(
     default= '',
     help   = 'any web address'
 )
+parser.add_argument(
+    '--insecure',
+    action = 'store_true',
+    dest   = 'insecure',
+    default= False,
+    help   = 'Indicates that the this message is from a trusted source, enabling https servers to accept self signed certificates'
```
Comment (danmcp) [inline bin/pfurl]: Bad sentence

**Comment by rudolphpienaar** [comment]
Is self.insecure a boolean?

If so, would you mind naming it b_insecure as per the conventions used in the rest of the code? Just keeps things consistent...

File: bin/pfurl (hunk 1 above)
Comment (iamemilio) [reply bin/pfurl]: suggested edit: 'allow insecure connections and self signed certificates over https'

**Comment by iamemilio** [comment]
@rudolphpienaar sure thing

**Comment by iamemilio** [comment]
fixed

**Comment by rudolphpienaar** [comment]
use ```self.dp.qprint``` instead of straight ```print```

**Pull request merged**